docs/src/main/paradox/docs/delta/benchmarks/v1.4.2.md

```diff
@@ -0,0 +1,276 @@
+# Benchmarks v1.4.2
+
+The tests were run against version 1.4.2 in January 2021 using @link:[Gatling](https://gatling.io/){ open=new } version 3.5.0.
+
+These are the most recent benchmarks. For older benchmarks of Nexus version 1.2.1 see @ref:[here](v1.2.1.md).
+
+# Environment
+
+The system was deployed on AWS EKS using the following basic configuration:
+
+*   Up to 12 compute nodes of 16 vCPU, 64 GB RAM
+*   [AWS EBS storage](https://aws.amazon.com/ebs/) - gp3, 10 000 IOPS, throughput 250 MB/s
+*   Nginx ingress controller
+*   1 Keycloak pod using its internal database H2 as storage, 5GB gp2 storage
+*   1 to 12 Cassandra pods each with 3.5 vCPU, 12 GB HEAP, 250 GB EBS storage
+*   1 Elasticsearch pod with 2 vCPU, 8 GB HEAP, 50GB gp2 storage
+*   1 Blazegraph pod with 2 vCPU, 4 GB HEAP, 50GB gp2 storage
+*   1 Prometheus pod and 1 Grafana pod, 50GB gp2 storage
+
+The services were deployed as follows:
+
```

Comment:
I would remove these lines, add the delta config next to the others and order them as follows:
- delta
- cassandra
- elastic
- bg
- keycloak
- prometheus
- nginx

Reply:
Change done and charts exported in PNG format.